e2e/nomostest/ntopts/multi_repo.go

```diff
@@ -47,6 +50,9 @@ type MultiRepo struct {
 
 	// SkipNonLocalGitProvider will skip the test if run with a GitProvider type other than local.
 	SkipNonLocalGitProvider bool
+
+	// NSReconcilerPolicyRules will grant a list of PolicyRules to NS reconcilers
+	NSReconcilerPolicyRules []rbacv1.PolicyRule
```

Comment:
I don't live having NS- here and also Namespace- in the same struct. Maybe this could be NamespaceAccess or RepoSyncPermissions or something?

Reply:
Updated, PTAL